keras_nlp/applications/bert.py

```diff
@@ -66,118 +67,118 @@ class BertModel(keras.Model):
     def __init__(
         self,
         vocab_size,
-        num_layers=12,
-        hidden_size=768,
-        dropout=0.1,
-        num_attention_heads=12,
-        inner_size=3072,
+        num_layers,
+        hidden_size,
+        num_attention_heads,
+        inner_size,
         inner_activation="gelu",
```

Comment:
I think this is technically incorrect for the BERT models, we need `tf.keras.activations.gelu(x, approximate=True)`. I don't think "approx_gelu" is a string that keras will recognize, but maybe we should set that as the default here, and just check for that string below in init?

Reply:
What do you think about not exposing this as a param @mattdangerw? Doesn't seem like something many users will care about, and then we don't have to serialize whatever crazy option they do choose.

Reply:
This sounds good! Let's not expose activation or initializer for now.